lib/ansible/modules/database/postgresql/postgresql_owner.py

```diff
@@ -0,0 +1,517 @@
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'supported_by': 'community',
+    'status': ['preview']
+}
+
+DOCUMENTATION = r'''
+---
+module: postgresql_owner
+short_description: Change an owner of PostgreSQL database object
+description:
+- Change an owner of PostgreSQL database object.
+- Also allows to reassign the ownership of database objects owned by a database role to another role.
+- For more information about REASSIGN OWNED BY command
+  see U(https://www.postgresql.org/docs/current/sql-reassign-owned.html).
+version_added: '2.8'
+
+options:
+  new_owner:
+    description:
+    - Role (user/group) to set as an I(obj_name) owner.
+    type: str
+    required: yes
+  obj_name:
+    description:
+    - Name of a database object to change ownership.
+    - Mutually exclusive with I(reassign_owned_by).
+    type: str
+  obj_type:
+    description:
+    - Type of a database object.
+    - Mutually exclusive with I(reassign_owned_by).
+    type: str
+    required: yes
+    choices: [ database, function, matview, sequence, schema, table, tablespace, view ]
+    aliases:
+    - type
+  reassign_owned_by:
+    description:
+    - The list of role names. The ownership of all the objects within the current database,
+      and of all shared objects (databases, tablespaces), owned by this role(s) will be reassigned to I(owner).
+    - Pay attention - it reassignes all objects owned by this role(s) in the I(db)!
+    - If role(s) exists, always returns changed True.
+    - Cannot reassign ownership of objects that are required by the database system.
+    - For more information see U(https://www.postgresql.org/docs/current/sql-reassign-owned.html).
+    - Mutually exclusive with C(obj_type).
+    type: list
+  fail_on_role:
+    description:
+    - If C(yes), fail when I(reassign_owned_by) role does not exist.
+      Otherwise just warn and continue.
+    - Mutually exclusive with I(obj_name) and I(obj_type).
+    default: yes
+    type: bool
+  db:
+    description:
+    - Name of database to connect to.
+    type: str
+    aliases:
+    - login_db
+  port:
+    description:
+    - Database port to connect.
+    type: int
+    default: 5432
+    aliases:
+    - login_port
+  login_user:
+    description:
+    - User (role) used to authenticate with PostgreSQL.
+    type: str
+    default: postgres
+  session_role:
+    description:
+    - Switch to session_role after connecting.
+      The specified session_role must be a role that the current login_user is a member of.
+    - Permissions checking for SQL commands is carried out as though
+      the session_role were the one that had logged in originally.
+    type: str
+  login_password:
+    description:
+    - Password used to authenticate with PostgreSQL.
+    type: str
+  login_host:
+    description:
+    - Host running PostgreSQL.
+    type: str
+  login_unix_socket:
```

Comment:
This looks like it should to be moved to a documentation fragment, which can be shared by all postgres modules. (Especially since you already have `postgres_common_argument_spec()`.)

Reply:
I believe the plan is to do a single PR to address this in all Postgres modules

Reply:
ок, thank you, good idea! Actually it is in all Postgres modules. Let's discuss it after FF. I'll make a PR for this that will change all ;)

Reply:
Sounds good.